tests/test_cli.py

```diff
@@ -74,5 +74,6 @@ def test_cowbird_services_list_with_formats():
         assert "+===" in out_lines[2]
         assert "| Magpie" in out_lines[3]
         assert "| Geoserver" in out_lines[4]
-        assert "| Thredds" in out_lines[5]
-        assert "| Nginx" in out_lines[6]
+        assert "| Catalog" in out_lines[5]
+        assert "| Thredds" in out_lines[6]
+        assert "| Nginx" in out_lines[7]
```

Comment:
Is this order always guaranteed? 
It looks to be related to some insertion order from the config (or else?) which could break if any config gets added anywhere but at the end.

Reply:
It can indeed break each time the example config change, but easily fixed. We could still parse the config here, but then we compare two parsing methods...

Reply:
Starting from the first service, could we use `.startswith`?
say,

```
services = ["| Magpie", "|  Geoserver", ... ]
...
assert out_line[3].startswith(tuple(services))
assert out_line[4].startswith(tuple(services))
```
Maybe even run it in a loop and add a check to make sure there are no duplicates?

My setup is not up yet so I can't check it, but I assume that each element of the `out_line` list is a string, for each line of the table output?

If this would make the test harder/too hard to understand, maybe just add a comment to the test to specify the order is directly related to the example config file (and active services) so someone new to the project doesn't even have to think about it when the test fails.

Reply:
I think a single sort of the output by name could solve all issues. 
It is also more natural to read the CLI output if services are sorted by name.